csvqb/csvqb/models/cube/csvqb/components/attribute.py

```diff
@@ -8,10 +8,23 @@
 
 from csvqb.models.uriidentifiable import UriIdentifiable
 from .datastructuredefinition import ColumnarQbDataStructureDefinition
-from .codelist import QbCodeList, NewQbCodeList
 from csvqb.models.validationerror import ValidationError
-from csvqb.inputs import PandasDataTypes
-from csvqb.models.cube.csvqb.catalog import CatalogMetadata
+from csvqb.inputs import PandasDataTypes, pandas_input_to_columnar_str
+
+
+@dataclass
+class NewQbAttributeValue(UriIdentifiable):
```

Comment:
~I think this misnamed, and it took me a while to figure out why. To me `value` is a reserved word in my headspace for objects in triples, not the definition of the predicates storing the values of a component. We need to talk about this before I approve this. As always I might need this explaining to me if I'm misunderstanding the RDF Qb spec.~

Is this basically an attribute codelist? If so, shouldn't the identifier be a uri_safe transformation of the label by default?

Reply:
Yes it is, and that's why the `get_identifier` method returns the label, it's used by the `uri_safe_identifier` property defined in `UriIdentifiable`